metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -334,7 +328,3 @@ class BspConnector(
     }
   }
 }
-
-object BspConnector {
-  final val BLOOP_SELECTED = "BLOOP"
```

Comment:
I realized we probably don't even need this, so I ripped it out of all the places and now just use `Bloop` from `BloopServers.name`. It simplifies it a bit.